playground/components_phylax.go

```diff
@@ -102,6 +102,7 @@ func (o *OpTalos) Run(service *Service, ctx *ExContext) {
 			"--ae.rpc_da_url", o.AssertionDA,
 			"--ae.rpc_url", "ws://localhost:8546",
 			"--ae.oracle_contract", o.OracleContract,
+			"--ae.rpc_store_db_path", "/data/rpc_store_db",
```

Comment:
Are you sure that your fix is correct?
Looking at the code it is actually using

```rs
    /// Path to the `assertion-executor` database.
    #[arg(long = "ae.db_path", default_value = "ae_database", env = "AE_DB_PATH")]
    pub db_path: PathBuf,
```

To store the relevant data
[12:28 Uhr](https://phylaxsystems.slack.com/archives/C06UPL4LQFQ/p1747650499415179?thread_ts=1747646731.169329&cid=C06UPL4LQFQ)
actually I cannot really find any usage of rpc_store_db